contracts/0.8.9/Burner.sol

```diff
@@ -10,6 +10,8 @@ import {SafeERC20} from "@openzeppelin/contracts-v4.4/token/ERC20/utils/SafeERC2
 import {Math} from "@openzeppelin/contracts-v4.4/utils/math/Math.sol";
 
 import {AccessControlEnumerable} from "./utils/access/AccessControlEnumerable.sol";
+import {Versioned} from "./utils/Versioned.sol";
```

Comment:
I suggest using https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/proxy/utils/Initializable.sol instead

Reply:
I would but:
- none of the 0.8.9 Core contracts use it 
- we need versioning anyway